benchmark/config_csv/blas3/trsm_square.csv

```diff
@@ -0,0 +1,6 @@
+l,u,n,n,64,64,1
```

Comment:
Can you add some non-square sizes as well? Also, would it be possible to put the TRSM and GEMM csv files into two separate folders? Maybe this restructuring change can be done in a separate PR?

Reply:
Done! #269